### PR TITLE
release: v1.35.0

### PR DESCRIPTION
**Pull request by tejal29**
Release v1.35.0!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,117 @@
...
+- Gaurav
+- Halvard Skogsrud
+- Jeremy Lewi
+- Marlon Gamez
+- Max Brauer
+- Nick Kubala
+- Pablo Caderno
+- Rouan van der Ende
+- Tejal Desai
+- dependabot[bot]
```
Comment (briandealwis) [inline CHANGELOG.md:112]: we don't need to thank dependabot

**Review by briandealwis — APPROVED** [review]
One nit

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,117 @@
...
+**Windows**
+https://storage.googleapis.com/skaffold/releases/v1.35.0/skaffold-windows-amd64.exe
+
+**Docker image**
+`gcr.io/k8s-skaffold/skaffold:v1.35.0`
+
+Note: This release comes with a new config version, `v2beta26`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+Highlights:
+* Skaffold's native support for `ko` builder for golang projects. Please try it out and let us [know](https://skaffold.dev/docs/pipeline-stages/builders/ko/)
```
Comment (nkubala) [inline CHANGELOG.md:23]:
```suggestion
* Skaffold now natively supports building Go projects with [`ko`](https://github.com/google/ko)! Check out the [documentation](https://skaffold.dev/docs/pipeline-stages/builders/ko/) for more information on how it works with Skaffold projects.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,117 @@
...
+* Skaffold performs status-check for stateful sets [#6828](https://github.com/GoogleContainerTools/skaffold/pull/6828)
```
Comment (nkubala) [inline CHANGELOG.md:24]:
```suggestion
* Skaffold now performs status-check for `StatefulSet` resources
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,117 @@
...
+- Halvard Skogsrud
+- Jeremy Lewi
+- Marlon Gamez
+- Max Brauer
+- Nick Kubala
+- Pablo Caderno
+- Rouan van der Ende
+- Tejal Desai
+- dependabot[bot]
+- jrcast
```
Comment (nkubala) [inline CHANGELOG.md:113]:
```suggestion
- Tejal Desai
- jrcast
```

**Review by nkubala — COMMENTED** [review]
conventional commit messages are definitely nice, but they don't really make our patch notes look very good....maybe we should remove the prefixes for the patch notes?

**Review by aaron-prindle — APPROVED** [review]
LGTM

**Pull request merged**